source/module.ts

```diff
@@ -17,11 +18,13 @@ export function formStoreFactory(ngRedux: NgRedux<any>) {
     ReactiveFormsModule,
   ],
   declarations: [
-    Connect,
+    ConnectTemplateDirective,
+    ConnectReactiveDirective,
     ConnectArray,
   ],
   exports: [
-    Connect,
+    ConnectTemplateDirective,
+    ConnectReactiveDirective,
```

Comment:
I don't see how, as the source of the form has now changed between the two. Maybe we can keep the template class named "connect" so that class stays the same for people plugged in for templates driven forms? Then, at least to me, the name of the class is less declarative on what it does but may be an option.